src/Features/Onboarding.php

```diff
@@ -633,12 +632,12 @@ public static function has_woocommerce_support( $theme ) {
 	}
 
 	/**
-	 * Append dynamic product data from API.
+	 * Get dynamic product data from API.
 	 *
 	 * @param array $product_types Array of product types.
 	 * @return array
 	 */
-	public static function append_product_data( $product_types ) {
+	public static function get_product_data( $product_types ) {
```

Comment:
@louwie17 Semantically, `append` typically implies that we're modifying data in place.  I don't think that's what this function is really doing; it's getting data from one array and returning a new array.

Even though it might appear that we're appending (or modifying) the original array, PHP is only passed by reference until written to (copy on write) so it's actually a new copy of the original array that we're returning.

I went ahead and created a new variable in https://github.com/woocommerce/woocommerce-admin/pull/8235/commits/016f0ca0eaa8bf4bfbca1d0b318c3e6ccc7f701e to make what's happening a bit more clear.  That said, I'm not overly opinionated on the name change, just thought it would be semantically more accurate for what's happening here.